loky/backend/reduction.py

```diff
@@ -113,7 +66,7 @@ def _rebuild_partial(func, args, keywords):
     return functools.partial(func, *args, **keywords)
 
 
-register(functools.partial, _reduce_partial)
+_dispatch_table[functools.partial] = _reduce_partial
```

Comment:
Let's revert this change and similar registrations to use the public `register` API instead.